xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingMarketDataService.java

```diff
@@ -170,6 +170,8 @@ public BinanceStreamingMarketDataService(
                     .originalAmount(rawTrade.getQuantity())
                     .instrument(currencyPair)
                     .price(rawTrade.getPrice())
+                    .makerOrderId(String.valueOf(rawTrade.getSellerOrderId()))
```

Comment:
maker or taker depends on if the limit order was a buy or sell, seller is not always a maker and buyer is not always a taker

Reply:
Thanks for the review, I corrected the code in respect of trade.isBuyerMarketMaker()